src/swap.c

```diff
@@ -2,43 +2,78 @@
 /* Swaps strings by swapping pointers */
 void swap_strings(char **str1_ptr, char **str2_ptr)
 {
-char *temp = *str1_ptr;
-*str1_ptr = *str2_ptr;
-*str2_ptr = temp;
+	char *temp = *str1_ptr;
+	*str1_ptr = *str2_ptr;
+	*str2_ptr = temp;
 }
-void swap_strings_deprecated(char *str1, char *str2)
+//function that return the smaller lenght of the compared strings
+int str_len_compare(char *str, char *str1)
```

Comment:
This function should return a bool 
If the length of str is greater than the length of str1  => return true / if not return false